lld/ELF/InputFiles.cpp

```diff
@@ -1702,6 +1703,38 @@ static uint8_t getOsAbi(const Triple &t) {
   }
 }
 
+namespace dtlto {
+// Check if an archive file is a thin archive.
+bool isThinArchive(Ctx &ctx, StringRef archiveFilePath) {
```

Comment:
`dtlto::isThinArchive` https://llvm.org/docs/CodingStandards.html#use-namespace-qualifiers-to-implement-previously-declared-functions

Reply:
Thanks. However, I think that these should just be static functions now and the namespace is unnecessary. 

It would be interesting to get your ideas about this area of the code. Currently, `isThinArchive` checks the magic number at the start of the archive file to determine if it is a thin archive - that is because: 1. ELF LLD doesn't record whether an archive is thin or not and 2. I wanted to add the minimum code to LLD for now and then address this properly this once the design of DTLTO was agreed. Perhaps the archive type could be recorded in `InputFile`s?

Reply:
These are static functions now.